.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -1,43 +1,43 @@
-name: Feature Request
-description: Requesting a new feature or changes to an existing feature
-title: '[FEATURE REQUEST]: <feature name>'
-labels: [enhancement]
-body:
-  - type: markdown
-    attributes:
-      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
-
-  - type: checkboxes
-    attributes:
-      label: I have searched through the issues and didn't find my problem.
-      options:
-        - label: Confirm
-          required: true
-
-  - type: textarea
-    id: description
-    attributes:
-      label: Description
-      description: Write down a detailed description of the feature.
-    validations:
-      required: true
-
-    - type: textarea
-      id: context
-      attributes:
-        label: Context
-        description: Why is this change important to you? How would you use it? How can it benefit other users?
-      validations:
-        required: true
-
-  - type: textarea
-    id: possiblesolution
-    attributes:
-      label: Possible Solution
-      description: If you have an idea, please tell us what might solve the above problem and the benefits it provides.
-
-  - type: textarea
-    id: extrainformation
-    attributes:
-      label: Additional information
-      description: Is there anything else we should know about this feature request?
+name: Feature Request
+description: Requesting a new feature or changes to an existing feature
+title: '[FEATURE REQUEST]: feature name'
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: "## Before you continue, please search our open/closed issues to see if a similar issue has been addressed."
+
+  - type: checkboxes
+    attributes:
+      label: I have searched through the issues and didn't find my problem.
```

You should use a string here to prevent YAML spotting `'` as the start of a string.
```suggestion
      label: "I have searched through the issues and didn't find my problem."
```